src/templates/client-details.tsx

```diff
@@ -1,21 +1,62 @@
+import { graphql } from 'gatsby';
 import React from 'react';
-
+import ClientPage from '../components/client-page/client-page';
+import { Grid, GridItem } from '../components/grid/grid';
 import Layout from '../components/layout/layout';
 import SEO from '../components/seo';
-import { PageTitle } from '../components/typography/typography';
-import { Grid, GridItem } from '../components/grid/grid';
 
-const ClientDetailsTemplate: React.FC = () => {
+interface ClientTemplateProps {
+  data: {
+    clientsJson: {
+      name: string;
+      industry: string;
+      description?: string;
+      tasks: string[];
+      techStack: string[];
+      teamSize?: number;
+      start: string;
+      currentInvestInDays?: number;
+      details?: string[];
+    };
+    markdownRemark: {
+      rawMarkdownBody: string;
+    };
+  };
+}
+
+const ClientDetailsTemplate: React.FC<ClientTemplateProps> = ({ data }) => {
   return (
     <Layout>
```

Comment:
You can add the property `siteTitleUrl={'/clients'}`. If you click on the header logo it will then redirect to the clients page and not the home page :)